<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>900</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="ExitBtn">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>800</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>10</y>
        <width>881</width>
        <height>38</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Ravie</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>LIBPRO</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Account Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="AccountEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PassEdit">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="Loginbtn">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Registerbtn">
             <property name="text">
              <string>Register</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="LogOut">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>800</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>700</width>
        <height>270</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="BookShow">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>310</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BookShow_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>310</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BookShow_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>310</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BookShow_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>310</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BookShow_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>310</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BookShow_6">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>20</y>
         <width>310</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BookShow_7">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>60</y>
         <width>310</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BookShow_8">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>100</y>
         <width>310</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BookShow_9">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>140</y>
         <width>310</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BookShow_10">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>180</y>
         <width>310</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>230</y>
         <width>661</width>
         <height>32</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>HOT BOOK!</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>NEW BOOK!</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>OLD GOOD BOOK!</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>450</y>
        <width>700</width>
        <height>300</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>698</width>
         <height>298</height>
        </rect>
       </property>
       <widget class="QLabel" name="BookName">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>660</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
       <widget class="QLabel" name="BookAuthor">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>660</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Author:</string>
        </property>
       </widget>
       <widget class="QLabel" name="BookType">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>660</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
       <widget class="QLabel" name="BookRate">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>660</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Rate:</string>
        </property>
       </widget>
       <widget class="QLabel" name="BookDes">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>660</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string>Describe:</string>
        </property>
       </widget>
       <widget class="QLabel" name="BookID">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>660</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>ID Book:</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>340</width>
        <height>150</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>331</width>
         <height>178</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="Name">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ID">
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="AccName">
          <property name="text">
           <string>Account:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ChangeInfo">
          <property name="text">
           <string>Full Info</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="OK">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>User Info</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AdminTab">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Admin </string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddCart">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>510</y>
        <width>91</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add cart</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Cart">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>120</y>
        <width>91</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Cart</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Search">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>120</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LibrarianTab">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>30</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Librarian</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QPushButton" name="Reader_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Reader</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>331</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="Name_4">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ID_4">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="AccName_4">
         <property name="text">
          <string>Account:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="AddBook">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>80</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>AddBook</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DeleteBook">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>80</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>DeleteBook</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SearchBook">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>80</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QListWidget" name="RequiredBook">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>241</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="RequiredBook1">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>200</y>
        <width>231</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Accept">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>290</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Accept</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="WatchBook_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Watch Book</string>
      </property>
     </widget>
     <widget class="QPushButton" name="WatchBook_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>260</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>511</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>320</y>
        <width>571</width>
        <height>241</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>569</width>
         <height>239</height>
        </rect>
       </property>
       <widget class="QLabel" name="BookName_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>660</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
       <widget class="QLabel" name="BookAuthor_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>660</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Author:</string>
        </property>
       </widget>
       <widget class="QLabel" name="BookType_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>660</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
       <widget class="QLabel" name="BookRate_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>660</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Rate:</string>
        </property>
       </widget>
       <widget class="QLabel" name="BookDes_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>660</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string>Describe:</string>
        </property>
       </widget>
       <widget class="QLabel" name="BookID_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>660</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="Back">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>600</width>
        <height>700</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>591</width>
         <height>681</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Acc</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QListWidget" name="listWidget"/>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>Reset Password</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="Exit">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>800</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>340</width>
        <height>150</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>331</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="Name_2">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ID_2">
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="AccName_2">
          <property name="text">
           <string>Account:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="Reader">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Reader</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
